### PR TITLE
[#159626572] Support HOST option for HTTP HEADER to PagoPa

### PR DESCRIPTION
**Pull request by LsKoder**
WIP: I'm testing it sniffing packets on my local env.

### CODE REVIEW

File: src/App.ts [hunk 1]
```diff
@@ -38,7 +38,11 @@ export async function startApp(config: Configuration): Promise<http.Server> {
     await PPTPortClient.createPagamentiTelematiciPspNodoClient({
       endpoint: `${config.PAGOPA.HOST}:${config.PAGOPA.PORT}${
         config.PAGOPA.WS_SERVICES.PAGAMENTI
-      }`
+      }`,
+      // Custom HTTP headers to be sent on WSDL requests
+      wsdl_headers: {
```
Comment (cloudify) [inline src/App.ts]: make it optional, like

```ts
wsdl_headers: config.PAGOPA.HOST_HEADER ? { Host: config.PAGOPA.HOST_HEADER } : { }
```

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#159626572](https://www.pivotaltracker.com/story/show/159626572): Aggiungere l'header HOST nelle chiamate HTTP verso PagoPa

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/App.ts (hunk 1 above)
Comment (LsKoder) [reply src/App.ts]: It's optional yet. Check the type declaration at line 106:
t.partial({
    HOST_HEADER: t.string
  })

Comment (cloudify) [reply src/App.ts]: yes but when HOST_HEADER is not defined you're still passing it to the soap library as the Host header!!!

Comment (LsKoder) [reply src/App.ts]: Nope. Client Header is set just if it's defined:

https://github.com/teamdigitale/italia-pagopa-proxy/blob/Add_PagoPa_Host_Header/src/utils/SoapUtils.ts#L22

**Pull request merged**